src/main/java/frc/robot/Robot.java

```diff
@@ -531,6 +570,78 @@ public void turnControlPanel() {
       m_controlPanelSpinAmount -= 1;
   }
 
+  private void processImage() {
+    m_sink.grabFrame(m_sourceMat);
+    if(doVisionProcessing) {
+      if(!m_sourceMat.empty()) {
+        // Crop out the uneccessary parts of the image
+        Mat m_subMat = new Mat(m_sourceMat, scanArea);
+
+        // Converts m_sourceMat to hsv color
+        Imgproc.cvtColor(m_subMat, m_hsvMat, Imgproc.COLOR_BGR2HSV);
+
+        // Filters for only the color of the tape
+        Core.inRange(m_hsvMat, hsvLow, hsvHigh, m_filteredMat);
+
+        // Find contours
+        Imgproc.findContours(m_filteredMat, contours, new Mat(),
+          Imgproc.RETR_TREE, Imgproc.CHAIN_APPROX_SIMPLE);
+
+        if(contours.size() > 0) {
+          MatOfPoint largestContour = contours.get(0);
+          double largestContourSize = Imgproc.contourArea(largestContour);
+          for(MatOfPoint contour : contours) {
+            double size = Imgproc.contourArea(contour);
+            if(size > largestContourSize) {
+              largestContour = contour;
+              largestContourSize = size;
+            }
+          }
+          contours.clear();
+          contours.add(largestContour);
+
+          Rect bounds = Imgproc.boundingRect(largestContour);
+          Point center = new Point(bounds.x + (bounds.width/2), bounds.y);
+          if(points.size() >= 4) {
+            points.remove(0);
+          }
+          points.add(center);
+          
+          center = new Point(0, 0);
+          for(Point point : points) {
+            center.x += point.x;
+            center.y += point.y;
+          }
+
+          center.x /= points.size();
+          center.y /= points.size();
+          center.y += 120;
```

Comment:
If this is a physical height (e.g. IRL height from x to y), add a constant for it. If it's something else (like an arbitrary amount of pixels), add a comment for that.

Reply:
This has been changed to pull directly from where it is derived from, `Constants.scanArea.y`.